runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -8140,11 +8227,22 @@ const char* am_reqStr(c_nodeid_t tgtNode, amRequest_t* req, size_t reqSize) {
     }
     break;
 
+  case am_opFAMOResult:
+    len += snprintf(buf + len, sizeof(buf) - len,
+                    ", obj %p, object %s"
+                    ", ofiType %s, sz %d",
+                    req->famo_result.obj,
+                    DBG_VAL(&req->famo_result.object, req->famo_result.ofiType),
+                    amo_typeName(req->famo_result.ofiType), 
```

Comment:
D'oh! Of course you need the `ofiType` of the result for debug output! Ha ha, what am I thinking? Okay, so let me adjust my previous feedback here, and say "You could put the `ofiType` member of the `am_opFAMOResult` request under `#ifdef CHPL_COMM_DEBUG` if you wanted to. I'm not sure it'll make a whole lot of difference, though. We probably couldn't measure the performance effect, and the additional ifdefs will clutter the code. So maybe ignore all my earlier feedback and just leave it there.